src/app/options/options.component.ts

```diff
@@ -1,18 +1,19 @@
-import { Component, OnInit } from '@angular/core';
```

Comment:
Seeing this now in its fuller form, it seems to make more sense to combine the input and output component into the options component and rename it.  This will DRY up your code and reduce the amount of code needing to be maintained. Please integrate the tests into the existing options test file as well.